src/mudlet.pro

```diff
@@ -715,7 +715,8 @@ LUA.files = \
     $${PWD}/mudlet-lua/lua/LuaGlobal.lua \
     $${PWD}/mudlet-lua/lua/Other.lua \
     $${PWD}/mudlet-lua/lua/StringUtils.lua \
-    $${PWD}/mudlet-lua/lua/TableUtils.lua
+    $${PWD}/mudlet-lua/lua/TableUtils.lua \
+    $${PWD}/mudlet-lua/lua/utf8_filenames.lua
```

Comment:
If we add the file to the resources, do we really need it listed here?

Reply:
No, but @SlySven will want it show up in Qt Creator...

Reply:
Then you'll probably need it to cmake as well.

Reply:
Not sure we have to do it in cmake as well for it to show in Qt Creator. This file doens't need to be installed.

I had a look through it but couldn't find anything on this subject.

Reply:
Right... cmake loops over lua files. I forgot.

But we still have https://github.com/Mudlet/Mudlet/blob/development/CMakeLists.txt#L217 (which produces a warning in the CI job). @SlySven remind me why that warning/check is in there in the first place?